Samples/openapi-v2/1a-code-generation-minimal/pétstöre.json

```diff
@@ -15,26 +15,16 @@
   },
   "host": "petstore.swagger.io",
   "basePath": "/v2",
```

Comment:
I think we should exclude all the files in the Samples folder from being formatted, the choices that Prettier makes seems to hurt readability and expected formatting somewhat here (and in the YAML files too)

Reply:
ok I can exclude those folders